.github/workflows/ci-kube-prometheus.yml

```diff
@@ -0,0 +1,133 @@
+# Copyright Broadcom, Inc. All Rights Reserved.
+# SPDX-License-Identifier: APACHE-2.0
+
+name: '[CI/CD] Special CI for kube-prometheus'
+on: # rebuild any PRs and main branch changes
+  pull_request_target:
+    types:
+      - opened
+      - reopened
+      - synchronize
+    branches:
+      - main
+      - bitnami:main
+    paths:
+      - 'bitnami/kube-prometheus/**'
+# Remove all permissions by default
+permissions: {}
+# Avoid concurrency over the same PR
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number }}
+jobs:
+  update-prometheus-rules:
+    runs-on: ubuntu-latest
+    name: Automatically update default Prometheus rules based on upstream
+    permissions:
+      contents: write
+    outputs:
+      result: ${{ steps.update-prometheus-rules.outputs.result }}
+    steps:
+      - name: Checkout bitnami/charts
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
+          token: ${{ secrets.BITNAMI_BOT_TOKEN }}
+          path: charts
+      - name: Setup git configuration
+        run: |
+          cd $GITHUB_WORKSPACE/charts
+          git config user.name "Bitnami Containers"
+          git config user.email "bitnami-bot@vmware.com"
+      - name: Install helm
+        run: |
+          HELM_TARBALL="helm-v3.8.1-linux-amd64.tar.gz"
+          curl -SsLfO "https://get.helm.sh/${HELM_TARBALL}" && sudo tar xf "$HELM_TARBALL" --strip-components 1 -C /usr/local/bin
+      - id: update-rules
+        name: Update Prometheus rules
+        env:
+          GITHUB_TOKEN: "${{ github.token }}"
```

Comment:
Why? this is the default behaviour

Reply:
Copy-pasted from CI workflow, I can remove it if needed